docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.8"
+
+services:
+  server:
+    build: .
+    environment:
+      IMS_DB_HOST_NAME: mysql
+      IMS_DB_DATABASE: ims
+      IMS_DB_USER_NAME: ims
+      IMS_DB_PASSWORD: ims
```

Comment:
Not a blocker. 
I've started a PR here https://github.com/chaoticbear/ranger-ims-server/pull/1 with a few extras like keeping the environment vars in a separate file to avoid git having to track these variable changes. I may do a follow up on this to add that in.

Reply:
👍 
I have the conf directory mounted in; I think it isn't parsing the conf file (if I'm reading the code correctly that's only possible by command line setting, so I'd have to override the CMD) -- it could just point at a conf ims yaml

Reply:
I did the above (as I realized trying other things that we definitely need to set the Admin users); it's still required to set IMS_DIRECTORY as this is set as an ENV in the docker file, but the rest of these are removed (and set into a custom conf sample in `conf/imsd-docker-compose-sample.conf`)